### ISSUE TITLE
Specifying custom localIdentName

### ISSUE THREAD

**Issue by mikkpokk** [JeffreyWay/laravel-mix#3123]
-   Laravel Mix Version: 6.0.34 (`npm list --depth=0`)
-   Node Version (`node -v`): 16.0.0
-   NPM Version (`npm -v`): 7.19.1
-   OS: macOS Big Sur 11.6

### Description:

By default class names are compiled as follows: `_2SDFUjk-ik2BVpsnbsG_nm`. However, I believe most of us are interested specifying custom local ident name for ex. `[local]_[hash:base64:5]`. Unfortunately, Laravel Mix 6 documentation is not describing how to do so. Is it possible at all?

**Comment by thecrypticace** [comment]
Took me a second to figure out what this was about. `localIdentName` is for css-loader. No there's no easy way to configure this currently.

**Comment by thecrypticace** [comment]
This was released in v6.0.38:

```js
mix.options({
  cssModuleIdentifier: '[name]_[local]__[hash:base64:5]'
})
```
